app/src/main/java/com/arctouch/codechallenge/app/CodeChallenge.kt

```diff
@@ -0,0 +1,15 @@
+package com.arctouch.codechallenge.app
+
+import android.app.Application
+import org.koin.android.ext.android.startKoin
+import org.koin.log.EmptyLogger
+
+class CodeChallenge :Application(){
```

Comment:
Just a code convention tip: Name your class suffixing their context, even for `Application`.

Reply:
Ok, I accustomed this way because is the pattern in my current company use the App name in Application classes.